src/lib/GameStats.ts

```diff
@@ -1,7 +1,7 @@
 import * as uuid from 'uuid/v4';
 import { KQStream, Character, PlayerKill } from './KQStream';
 
-type StatisticType = 'kills' | 'deaths';
+type StatisticType = 'kills' | 'queen_kills' | 'other_kills' | 'deaths';
```

Comment:
i ideally wanted to make the "kills" statistic be a rich object have nested attributes... but I punted... this is probably an area to improve.

Reply:
Yea, something like "type of kill". Could indicate that a queen, warrior, or drone was killed.